rocksdb_admin/tests/admin_handler_test.cpp

```diff
@@ -157,6 +157,29 @@ TEST(AdminHandlerTest, CheckDB) {
   thread->join();
 }
 
+TEST(AdminHandlerTest, PartitionID) {
+  EXPECT_EQ(std::system("rm -rf /tmp/meta_db"), 0);
+
+  auto db_manager = std::make_unique<admin::ApplicationDBManager>();
+  admin::AdminHandler handler(std::move(db_manager),
+                              admin::RocksDBOptionsGeneratorType());
+
+  std::string db_name;
+  int partition;
+
+  db_name = "test_db_00000";
+  partition = handler.getPartitionIDFromDBName(db_name);
+  EXPECT_EQ(partition, 0);
+
+  db_name = "test_db_00030";
```

Comment:
fwiw, helix will change the db name to be the format for rocksdb, so it'd send test_db00030 in the thrift request

Reply:
Fixed.